### PR TITLE
[ticket/15110] Meta Tags Generating Function Added

### PR DESCRIPTION
**Pull request by naveen17797**
Checklist:

- [ ] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [ ] Tests pass
- [ ] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [ ] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15110

Meta tags function now utilizes the variables which are obtained from the existing queries unlike the old PR ,those variables are generated from new queries

### CODE REVIEW

**Comment by DavidIQ** [comment]
Hi there,

This will also fail the build just like your other pull request. If you're going to make further contributions please note the following:

1. Your tabs and spaces are all off.
2. You have unnecessary spacing changes throughout your commits.
3. All of your commit messages are wrong. The proper format is in the link in the above pull request description.
4. You added 3 additional queries for information that is already being retrieved in viewtopic.

**Comment by naveen17797** [comment]
hi,
sorry for making this mistake.I will continue my contribution
according to format as you mentioned above.
i have some queries regarding this project
1. should i make this meta tags generating function as extension?
2.will travis ci build fail if there is additional spaces?

**Comment by DavidIQ** [comment]
1. Not necessarily. The developers will decide that.
2. No, but this pull request will not get accepted/merged with all that useless spacing.

**Comment by naveen17797** [comment]
hey,
thanks for responding.I will close this PR and Start contributing
according to the proper commit format.

File: phpBB/includes/functions_display.php [hunk 1]
```diff
@@ -1715,3 +1716,38 @@ function phpbb_sort_last_active($first, $second)
...
+<meta property='og:image' content='$phpbb_domain/$phpbb_forum_image' />
+
+<!--meta tags for twitter-->
+<meta name='twitter:card' content='summary_large_image'>
+<meta name='twitter:site' content='@$phpbb_domain'>
+<meta name='twitter:creator' content='$phpbb_topic_author'>
+<meta name='twitter:title' content='$phpbb_topic_title'>
+<meta name='twitter:description' content='$phpbb_topic_description'>
+<meta name='twitter:image' content='$phpbb_domain/$phpbb_forum_image'>
+
```
Comment (hanakin) [inline phpBB/includes/functions_display.php]: I can however say that this is not going to be merged, as a rule no HTML should be included in PHP or JS files. We try to keep a clean division of labor you might consider creating template variables for the content and a template file for the tags

**Comment by CHItA** [comment]
duplicate of #4733

**Pull request closed without merging**